conans/model/manifest.py

```diff
@@ -17,30 +15,22 @@ def discarded_file(filename):
 
 def gather_files(folder):
     file_dict = {}
-    symlinks = {}
-    for root, dirs, files in walk(folder):
+    symlinked_folders = {}
+    for root, dirs, files in walk(folder, followlinks=True):
```

Comment:
Are we sure this will not package system files? For example if a package defines some symlink from a folder to ``/usr/include``, then this ``followlinks`` might iterate the files there?

Reply:
I think this should avoid that:
```
if os.path.islink(root):
            dirs[:] = []
```
but I will make sure

Reply:
anyway the `followlinks=True` is not needed so I'm removing it.